verify-cborld-presentation/src/pages/scan.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 - MATTR Limited
+ * All rights reserved
+ * Confidential and proprietary
+ */
+
+import { Box } from "@material-ui/core";
+import React from "react";
+import { useHistory } from "react-router";
+import QrReader from "react-web-qr-reader";
+import styled from "styled-components";
+
+import { Description, DescriptionTitle } from "../components";
+
+import { getQrData } from "./qr";
+
+export interface ScanPageProps {
+  onScannedQrCodeChange: (scannedQrCode: string) => void;
+}
+
+export const ScanPage: React.FC<ScanPageProps> = (props) => {
+  const history = useHistory();
+
+  const onScan = async (data: any): Promise<void> => {
+    if (typeof data.data === "string") {
+      console.log("scanned QR code", data.data);
```

Comment:
remove console.log

Reply:
For a sample app, probably OK...

Reply:
Yeah nice for someone to see what's happening